std/container/array.d

```diff
@@ -1585,6 +1585,19 @@ if (!is(Unqual!T == bool))
     ai.insertBack(arr);
 }
 
+@system unittest
```

Comment:
Is there a reason this can't be `@safe`?

Reply:
destructor std.container.array.Array!(MyClass).Array.~this is @system

Reply:
Could you please add a comment explaining that then? Thanks!

Reply:
Done.